### ISSUE TITLE
ignoreRegExpList start to fail after update to Version 4.0.40

### ISSUE THREAD

**Issue by paulrm** [streetsidesoftware/vscode-spell-checker#4252]
My usual config

```
  "cSpell.ignoreRegExpList": [
    "/```[\\s\\S]*?```/",             // Ignore fenced code blocks
    "`[^`]*`"                         // Ignore inline code blocks
  ],
  "cSpell.languageSettings": [
    {
        "languageId": "markdown",
        "ignoreRegExpList": [
            "/```[\\s\\S]*?```/",    // Ignore fenced code blocks
            "`[^`]*`"                // Ignore inline code blocks
        ]
    }
  ],

```

but after update  to 4.0.40, is reporting errors in code blocks
(also in pre-release 4.0.41)

How can debug this issue to provide more feedback?

VSCode
Version: 1.98.2 (Universal)
Commit: ddc367ed5c8936efe395cffeec279b04ffd7db78
Date: 2025-03-12T13:32:45.399Z (1 wk ago)
Electron: 34.2.0
ElectronBuildId: 11161602
Chromium: 132.0.6834.196
Node.js: 20.18.2
V8: 13.2.152.36-electron.0
OS: Darwin arm64 24.3.0

**Comment by paulrm** [comment]
False positive, please ignore
